CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v1.11.3
+
+- [RUMF-447]: Only collect first-contentful-paint if page is visible ([#361](https://github.com/DataDog/browser-sdk/pull/361))
+- ⬆️ upgrade wdio-\* ([#381](https://github.com/DataDog/browser-sdk/pull/381))
+- 🔊 add staging/prod deployment notifications in ops channels ([#379](https://github.com/DataDog/browser-sdk/pull/379))
+- ✅ try to fix flaky edge test ([#378](https://github.com/DataDog/browser-sdk/pull/378))
```

Comment:
IMO, we should only add PR that can impact customers.
We will have too much noise otherwise